packages/components/src/Menu/Menu.tsx

```diff
@@ -264,20 +264,34 @@ export interface ActionProps {
    */
   readonly icon?: IconNames;
 
+  /**
+   * Visual style for the action button
+   */
+  readonly buttonVariation?: "default" | "destructive";
```

Comment:
Can we update this to match our style in a few [other spots](https://github.com/GetJobber/atlantis/blob/17a49de3c78e2db92c4f775bc954b362ba8527ce/packages/components/src/DataList/DataList.types.ts#L342)? Just to be a boolean instead of a typed string.

Reply:
For sure! The original intention of the typed string was since we have other button variations - this would enable the door to be opened for that. If you feel this isn't needed, I can go ahead with the boolean!